sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClientBuilder.java

```diff
@@ -535,4 +549,12 @@ public TextAnalyticsClientBuilder serviceVersion(TextAnalyticsServiceVersion ver
         this.version = version;
         return this;
     }
+
+    private boolean isConsolidatedServiceVersion(TextAnalyticsServiceVersion serviceVersion) {
+        if (serviceVersion == null) {
+            serviceVersion = TextAnalyticsServiceVersion.V2022_04_01_PREVIEW;
```

Comment:
We should also make a note of the service version default in readme.

Reply:
yeah. WIll do it in the next documentation PR